qa/tasks/cephfs/test_subvolume.py

```diff
@@ -168,3 +172,33 @@ def test_subvolume_create_snapshot_inside_new_subvolume_parent(self):
 
         # clean up
         self.mount_a.run_shell(['rmdir', 'group/subvol2/dir/.snap/s2'])
+
+    def test_subvolume_vxattr(self):
+        """
+        To test presence and absence of ceph.dir.subvolume xattr on a subvolume
+        """
+        # create a subvolume
+        self._fs_cmd('subvolumegroup', 'create', self.fs.name, 'grp1')
+        self._fs_cmd('subvolume', 'create', self.fs.name, 'sv1', 'grp1')
+
+        # verify that ceph.dir.subvolume has value "1"
+        stdo = StringIO()
+        self.mount_a.run_shell(['getfattr', '-n', 'ceph.dir.subvolume',
```

Comment:
Can you use the helper functions here and below? See `mount.py` for more information